astropy/coordinates/earth.py

```diff
@@ -602,6 +603,30 @@ def get_itrs(self, obstime=None):
                                      for the location of this object at the
                                      default ``obstime``.""")
 
+    def get_gcrs(self, obstime):
```

Comment:
This is nominally a new feature and should be documented as such, but then we run into the awkwardness of "new feature" despite this being a bugfix.  I wonder if the way to go is to have it be private (e.g. `_get_gcrs`) for this PR but then add a follow-on (trivial) PR for 3.0 that just de-privatizes it and has a changelog entry mentioning it?

Reply:
👍